src/Synergism.ts

```diff
@@ -4011,8 +4014,11 @@ export const reloadShit = async (reset = false) => {
 
 window.addEventListener('load', () => {
     const ver = DOMCacheGetOrSet('versionnumber');
+    const addZero = (n: number) => {
+        return (n<10 ? '0' + n : n);
+    }
     if (ver instanceof HTMLElement) {
-        const textUpdate = !isNaN(lastUpdated.getTime()) ? ` [Last Update: ${lastUpdated.getHours()}:${lastUpdated.getMinutes()} UTC ${lastUpdated.getDate()}-${lastUpdated.toLocaleString('en-us', {month: 'short'})}-${lastUpdated.getFullYear()}].` : '';
+        const textUpdate = !isNaN(lastUpdated.getTime()) ? ` [Last Update: ${addZero(lastUpdated.getHours())}:${addZero(lastUpdated.getMinutes())} UTC ${addZero(lastUpdated.getDate())}-${lastUpdated.toLocaleString('en-us', {month: 'short'})}-${lastUpdated.getFullYear()}].` : '';
```

Comment:
Surprisingly(?) this is the easiest way to handle in javascript. fixes the bug of time showing up as 18:5 instead of 18:05, and if we're using 24-hour clock then technically we want a leading zero on the hour too. Why not also include the day.